ui/explore/src/components/ExploreManager/MetricsExplorer.tsx

```diff
@@ -38,10 +37,43 @@ function TimeSeriesPanel({ queries }: { queries: QueryDefinition[] }) {
 
   return (
     <Box ref={boxRef} height={height}>
-      <DataQueriesProvider definitions={definitions} options={{ suggestedStepMs }}>
-        <TimeSeriesChart.PanelComponent
-          contentDimensions={{ width: width ?? PANEL_PREVIEW_DEFAULT_WIDTH, height }}
-          spec={{}}
+      <DataQueriesProvider definitions={definitions} options={{ suggestedStepMs, mode: 'range' }}>
+        <Panel
+          panelOptions={{
+            hideHeader: true,
+          }}
+          definition={{
+            kind: 'Panel',
+            spec: { queries: queries, display: { name: '' }, plugin: { kind: 'TimeSeriesChart', spec: {} } },
+          }}
+        />
+      </DataQueriesProvider>
+    </Box>
+  );
+}
+
+function MetricDataTable({ queries }: { queries: QueryDefinition[] }) {
+  const height = PANEL_PREVIEW_HEIGHT;
```

Comment:
IMHO the height should be increased in this case, we're quickly reaching the max height with just a few results here - and contrary to the Dashboard view, you cannot resize any panel height to adjust:

![image](https://github.com/perses/perses/assets/7058693/3c4861a7-0a7c-448a-913b-24c30a0cba8b)

On prom UI they went for infinite expand, I dont know if we want to go this way (it causes other navigation issues when you have multiple queries returning multiple results), but I think as a first version we can go with the current PANEL_PREVIEW_HEIGHT as min height and at least the triple of that for max height

Reply:
I'd like to have that control on the height of a panel but unfortunately the first element of the ``Panel`` is saying height=100% already. 

It forces me to give a value and I cannot know the height of the child component below ``Panel``. 

The best solution is to introduce some params to the ``Panel`` component to optionally disable this height 100% and give more control on min height / max height.